### DISCUSSION TITLE
Status der Verbindung flipped bei mqtt

### DISCUSSION THREAD

**Discussion by ronzle** [solectrus/solectrus#3318] [General]
Hi, 
Ich habe auf den mqtt collector umgestellt, da ich evcc, wegen des Solarueberstrom Laden nutzen will. 
Das ganze laeuft auch super, bis auf das ich die Gehaeusetemperatur nicht sehe ... 
Aber viel verwirrender, ist das die Verbindungsanzeige immer zwischen Verbunden und Nicht Verbunden springt. 
Kann man hier irgendwo eine Einstellungen machen, oder muss da was am MQTT Broker gesetzt werden ? 

Danke und Gruss
Ronny 

![solectrus_error](https://github.com/user-attachments/assets/eef43164-6f48-467f-8035-bac9c843adef)

**Comment by ledermann** [comment]
Das liegt daran, dass das Dashboard die Daten schneller wartet, als der MQTT-Collector sie in die InfluxDB schreibt.

Das Dashboard kennt die Variable `INFLUX_POLL_INTERVAL` (siehe https://github.com/solectrus/solectrus/wiki/Konfiguration#zugriff-auf-influxdb). In diesem Turnus (per Default 5 Sekunden) werden neue Daten erwartet. Werden diese nicht gefunden (mit einer gewissen Karenzzeit), dann heißt es "Keine Verbindung". 

Du musst also entweder `INFLUX_POLL_INTERVAL` hochsetzen oder dein MQTT-Setup so verändern, dass die Messwerte häufiger aktualisiert werden.

**Reply by ronzle** [reply]
Ok danke perfekt ! 
Danke fuer den wink in die richtige Richtung
